<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCadastroClientes_Cachorro</class>
 <widget class="QDialog" name="JanelaCadastroClientes_Cachorro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cachorro</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>187</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Raça:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_raca"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pelagem:*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_pelagem"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>75</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="cb_adestrado">
      <property name="text">
       <string>Adestrado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_castrado">
      <property name="text">
       <string>Castrado</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_cachorroInserir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Campo obrigatório</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cachorro:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cachorroAlterar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
